lib/repl.js

```diff
@@ -1257,12 +1257,13 @@ function defineDefaultCommands(repl) {
   });
 
   repl.defineCommand('help', {
-    help: 'Show repl options',
+    help: 'Print this help message',
     action: function() {
-      var self = this;
-      Object.keys(this.commands).sort().forEach(function(name) {
-        var cmd = self.commands[name];
-        self.outputStream.write(name + '\t' + (cmd.help || '') + '\n');
+      Object.keys(this.commands).sort().forEach((name) => {
+        const cmd = this.commands[name];
+        const spaces = ' '.repeat(name.length <= 9 ? 9 - name.length : 1);
```

Comment:
@silverwind Find the max length programmatically. User defined commands may have bigger name 😄